sdk/python/feast/infra/materialization/contrib/bytewax/bytewax_materialization_engine.py

```diff
@@ -82,6 +82,12 @@ class BytewaxMaterializationEngineConfig(FeastConfigBaseModel):
     mini_batch_size: int = 1000
     """ (optional) Number of rows to process per write operation (default 1000)"""
 
+    bytewax_replicas: int = 5
+    """ (optional) Number of process to spawn in each pods to handle a file in parallel"""
+
+    bytewax_worker_per_process: int = 1
```

Comment:
ah yeah you're right, looking forward to working on a separate PR

Reply:
> I don't believe that this setting, and the setting of the number of replicas will currently be respected.
> 
> I'd be happy to help with getting this fixed, but perhaps that should be the subject of a separate PR.

could you point out exactly where the setting is not respected? seems like the env of bytewax pods consume it correctly

Reply:
Great, would love to chat more, but by the way we're using bytewax 0.17.1 as of now

Reply:
by the way, in case the REPLICAS setting doesn't work, we still have at least 1 process and 1 worker per pods.

and the implementation of this PR still reduce memory footprint compares to master's one. should we process this first? @achals

Reply:
@sudohainguyen  I'd prefer to update the pr so only the changes we expect to actually make a difference are merged (specifically the arrow changes) - we can bump the dependency on bytewax and make any necessary changes for replicas in a separate pr

Reply:
agree, then I will narrow down the PR, to make bytewax settings unexposed from batch engine config

Reply:
done @achals